lib/course_planner/Mailer/main.ex

```diff
@@ -0,0 +1,6 @@
+defmodule CoursePlanner.Mailer.Main do
```

Comment:
Maybe just leave it as `CoursePlanner.Mailer`
The other mailer is `CoursePlanner.Coherence.Mailer` which is specific to coherence, or change the folder structure.
Folder naming should be downcased as well.